eurostag-step-up-transformer/src/main/java/eu/itesla_project/iidm/ddb/eurostag/EurostagDDB.java

```diff
@@ -7,34 +7,56 @@
 package eu.itesla_project.iidm.ddb.eurostag;
 
 import java.io.IOException;
+import java.nio.file.FileVisitOption;
 import java.nio.file.FileVisitResult;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.SimpleFileVisitor;
 import java.nio.file.attribute.BasicFileAttributes;
+import java.util.EnumSet;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  *
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 class EurostagDDB {
 
     private final Map<String, Path> generators = new HashMap<>();
-
+    private static final Logger LOGGER = LoggerFactory.getLogger(EurostagDDB.class);
     EurostagDDB(List<Path> ddbDirs) throws IOException {
         for (Path ddbDir : ddbDirs) {
+            if (Files.isSymbolicLink(ddbDir)) {
+                ddbDir = Files.readSymbolicLink(ddbDir);
+            }
```

Comment:
This code does not handle several link levels. You should probably add a function like this to resolve links:
```
    private static Path readSymbolicLink(Path link) throws IOException {
        Path path = link;
        while (Files.isSymbolicLink(link)) {
            path = Files.readSymbolicLink(path);
        }
        
        return path;
    }
```